ago-dic-2018/practicas/practica_1/Algoritmos.java

```diff
@@ -0,0 +1,31 @@
+
+package algoritmos;
+
+
+public class Algoritmos {
+
+    public static void main(String[] args) {
+
+        String[]nombres={"Juan","Juan","Pepe","Ana"};
+int n=0;
+String Nombrepopular="";
+         
+            for (int i = 0; i <nombres.length; i++) {
+            int contador=0;
+            
+             
+            for (int j = 0; j < nombres.length; j++) {
+                 if (nombres[i]==nombres[j]){
+                     contador++;
+                 }
+                 if(contador>n){
```

Comment:
esta evaluación la puedes hacer solo una vez, por cada nombre, afuera de este segundo for